app/assets/stylesheets/global.scss

```diff
@@ -4,4 +4,5 @@
   line-height: 45px;
   padding: 0 15px;
   color: #fff;
+  background-color: #CCCCCC;
```

Comment:
Color literals like `#CCCCCC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#CCCCCC` should be written as `#cccccc`